tslint.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
We will need to migrate our internal eslint profile to tslint. Let's note this as separate task.